src/reachy2_sdk/parts/hand.py

```diff
@@ -126,12 +140,29 @@ def is_off(self) -> bool:
         """Get compliancy of the hand"""
         return self._compliant
 
+    def is_motion_complete(self) -> bool:
+        """Get state of gripper movement"""
+        if not self._request_sent:
+            return True
+        else:
+            return self._request_reached
+
     def _update_with(self, new_state: HandState) -> None:
         """Update the hand with a newly received (partial) state received from the gRPC server."""
         self._present_position = new_state.present_position.parallel_gripper.position.value
         self._goal_position = new_state.goal_position.parallel_gripper.position.value
         self._opening = new_state.opening.value
         self._compliant = new_state.compliant.value
+        if self._request_sent:
+            if len(self._last_present_positions) < self._last_present_positions_queue_size:
+                pass
+            elif np.isclose(self._present_position, self._last_present_positions[-1], np.deg2rad(0.1)) and np.isclose(
```

Comment:
I'd make a simple function called check gripper movement (something like that). I'd be easier to understand what we are testing here.

Reply:
Done